front-end/src/components/charts/line-chart.jsx

```diff
@@ -1,44 +1,94 @@
-import React from "react";
+import React, {useEffect} from "react";
 import "chart.js/auto";
 import {Line} from "react-chartjs-2";
 import {ChartDiv} from "./line-chart.styles";
 import SaveButton from "../buttons/save-button";
 
 const LineChart = ({chartTitle, chartData}) => {
-	return (
-		<ChartDiv>
-			<Line
-				data={{
-					labels: [
-						"Red",
-						"Blue",
-						"Yellow",
-						"Green",
-						"Purple",
-						"Orange",
-					],
-					datasets: [
-						{
-							label: `${chartTitle} data`,
-							data: ["32", 91, 3, 5, 2, 3],
-							backgroundColor: ["#BAD4C6ff"],
-							borderColor: ["#2C4244ff", "#2D8840ff;"],
-							borderWidth: 1,
-						},
-					],
-				}}
-				options={{
-					plugins: {
-						title: {
-							display: true,
-							text: `${chartTitle}`,
-						},
-					},
-				}}
-			/>
-			<SaveButton />
-		</ChartDiv>
-	);
+  let datasets = [];
+  let labels = chartData.map((dataObj) => Math.floor(dataObj.delta));
+  if (chartTitle === "cpu") {
+    const userData = chartData.map((dataObj) => dataObj.user);
+    const systemData = chartData.map((dataObj) => dataObj.system);
+    datasets.push(
+      {
+        label: `user`,
+        data: userData,
+        backgroundColor: ["#BAD4C6ff"],
+        borderColor: ["#2C4244ff", "#2D8840ff;"],
+        borderWidth: 1,
+      },
+      {
+        label: `system`,
+        data: systemData,
+        backgroundColor: ["#72524a"],
+        borderColor: ["#ebcfc9", "#2D8840ff;"],
+        borderWidth: 1,
+      }
+    );
+  } else if (chartTitle === "eventloop") {
+    const eventloopLabels = Object.keys(chartData[0]).filter((key) => {
```

Comment:
Array.prototype.filter() expects a value to be returned at the end of arrow function.eslint[array-callback-return](https://eslint.org/docs/rules/array-callback-return)

`filter` should return true or false